torch/jit/__init__.py

```diff
@@ -708,5 +708,14 @@ def __dir__(self):
         keys = [key for key in keys if not key.isdigit()]
         return keys
 
+    def forward(self, inputs):
```

Comment:
This prevents the script from being a subset of what's valid in Python. `nn.ModuleList` doesn't have a forward, and trying to use it should result in an error. `_ConstModuleList` is its compiled counterpart, and so it should stay the same. A better solution to this would be to add `_ConstSequential` that subclasses from `_ConstModuleList` and additionally does implement `forward`

Reply:
Also, I think this method could have been compiled, right?

Reply:
Thank you! I've add _ConstSequential as the subclass of _ConstModuleList. What does it mean by "this method could have been compiled"?

Reply:
I meant that it could have been a script method. If someone marks the `Sequential` as const, then we know that the set of submodules won't change, so we can optimize the forward function by e.g. unrolling the loop.